docs/core/sdk.md

```diff
@@ -22,7 +22,7 @@ It contains the following components:
 
 1. The .NET Core Command Line Tools that are used to build applications
 2. .NET Core (libraries and runtime) that allow applications to both be built and run
-3. The `dotnet` driver for running the CLI commands as well as running applications
+3. The `dotnet` driver for running the [CLI commands](tools/index.md) as well as running applications
 
 
 ## Acquiring the .NET Core SDK
```

Comment:
This topic is in my to-do list to review: #781 

I agree. It has some of the same contents as the tools page, so we could link to that when talking about the CLI.